src/Sarif.UnitTests/FileDiffingTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT        
+// license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.IO;
+using Microsoft.CodeAnalysis.Sarif.Readers;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.CodeAnalysis.Sarif.UnitTests
+{
+    public class FileDiffingTests
+    {
+        protected string GenerateDiffCommand(string expected, string actual)
+        {
+            expected = Path.GetFullPath(expected).Replace(@"\", @"\\");
+            actual = Path.GetFullPath(actual).Replace(@"\", @"\\");
+
+            string beyondCompare = TryFindBeyondCompare();
+            if (beyondCompare != null)
```

Comment:
>beyondCompare [](start = 16, length = 13)

I know this is existing code, but... NO! NO! NO! This code means that a unit test could potentially give different results on machines where Beyond Compare is installed than on machines where it's not installed.

Reply:
Ok, ok, I see now that the result is only used in a message, telling the user how to see the differences between actual and expected files.

---
In reply to: [205981120](https://github.com/Microsoft/sarif-sdk/pull/974#discussion_r205981120) [](ancestors = 205981120)